src/content/protocols/aave.md

```diff
@@ -0,0 +1,888 @@
+
+### Core Lending & Borrowing
+
+This subsystem forms the core of Aave v3's borrow & lending features and keeps track of users' positions and related protocol state. It centers around the `Pool` contract that governs how debt in the system can be built up, collateralized and how positions can be liquidated. It includes Aave v3's `aTokens` and `variableDebtTokens`. All of these contracts are fully upgradeable through the _Aave Governance_ [subsystem](#aave-governance). An unintended (or malicious) contract upgrade can result in the loss of user funds, loss of unclaimed yield or otherwise materially affect the expected protocol performance.
+
+### Reserve Parameters
+
+The _Reserve Parameters_ subsystem is responsible for maintaining and updating critical risk parameters in the Aave v3 protocol. Among others, these include the set of enabled collateral and loan assets, their specific _liquidation loan-to-value_ ratios, _liquidation bonuses_. An upgradeable `PoolConfigurator` contract controls these parameters with permissions to implement changes delegated to different, specialized, multisig accounts such as the _Emergency Admin_ or _Risk Council_. A system of _Roles_, implemented using OpenZeppelin's ACL pattern, and _Stewards_ ensures that these multisig accounts can only implement changes in a limited range or over a certain time period. This system thus limits the risk of unintended updates through the multisig accounts. However, an upgrade of the `PoolConfigurator` contract could revert these safeguards thus reintroducing the risk of loss of funds, loss of unclaimed yield or a material change of the expected protocol performance.
+
+### Treasury, Aave Ecosystem Reserves and Rewards System
+
+This subsystem manages how fees are collected, spent and third-party rewards allocated and claimed by users. Fees taken across the Aave v3 protocol (and previous versions) are collected in the `TreasuryCollector`, a fully upgradeable contract. Through _Aave Governance_ and different (upgradeable) utility contracts these funds are distributed among service providers and ecosystem initiatives. Furthermore, third parties are able to distribute rewards, in their own ERC20 token, to Aave users through the `RewardsControllerManager` and `EmissionManager` contracts. Rewards are distributed continuously among eligible users but controlled by the _Aave Chan Initiative_ multisig account until claimed. The _Aave Chan Initiative_ multisig account can thus potentially be abused to withhold or steal distributed rewards resulting in a loss of unclaimed yield for users.
```

Comment:
each incentive can be placed to a new rewards controller multisig (as ACI) it's technically not given that it has to be ACI, but rather "social practice"

Reply:
discuss if additional "foreign" yield is in scope